### ISSUE TITLE
Grid lines for each day

### ISSUE THREAD

**Issue by nschloe** [opendata-stuttgart/feinstaub-map#55]
In the current graph plots

![24](https://user-images.githubusercontent.com/181628/36646184-937d39f6-1a74-11e8-8765-21feaf4fdec0.png)

it's next to impossible to distinguish between the individual days. It'd great to have grid lines at 12am every day. This makes it easier to tell if the daily average is smaller or larger than the legal limit of 50 µg/m^3.

**Comment by ricki-z** [comment]
We are limited to the abilities of grafana at this. But you could create graphs for yourself if you want, this is one of the advantages of open data ...

**Comment by nschloe** [comment]
Yeah I'm not gonna write my own map for this. It's just a feature request...
